man/sd_journal_get_fd.xml

```diff
@@ -151,18 +151,20 @@ else {
     <function>poll()</function> and
     <function>sd_journal_process()</function> into one.</para>
 
-    <para><function>sd_journal_reliable_fd()</function> may be used to
-    check whether the wakeup events from the file descriptor returned
-    by <function>sd_journal_get_fd()</function> are known to be
-    immediately triggered. On certain file systems where file change
-    events from the OS are not available (such as NFS) changes need to
-    be polled for repeatedly, and hence are detected only with a
-    certain latency. This call will return a positive value if the
-    journal changes are detected immediately and zero when they need
-    to be polled for and hence might be noticed only with a certain
-    latency. Note that there is usually no need to invoke this function
-    directly as <function>sd_journal_get_timeout()</function> on these
-    file systems will ask for timeouts explicitly anyway.</para>
+    <para><function>sd_journal_reliable_fd()</function> may be used to check whether the wake-up events from
+    the file descriptor returned by <function>sd_journal_get_fd()</function> are known to be quickly
+    triggered. On certain file systems where file change events from the OS are not available (such as NFS)
+    changes need to be polled for repeatedly, and hence are detected only with a considerable latency. This
+    call will return a positive value if the journal changes are detected quickly and zero when they need to
+    be polled for and hence might be noticed only with a considerable latency. Note that there is usually no
```

Comment:
Drop " and hence might be noticed only with a considerable latency"? It's just repeating what was already said.